doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -399,6 +399,16 @@ The following strings must not be used as attribute names:
 
 * Virtual attribute names (see specific section on "Virtual Attributes")
 
+## Metadata names restrictions
+
+The following strings must not be used as metadata names:
+
+* `ngsi:onArrival`, as it has an special use (see "Notification metadata marks" section).
```

Comment:
I have tried to explain at the cited section... https://github.com/telefonicaid/fiware-orion/pull/2518/files#diff-d5ec9d3a1874c68d6328577a86cd5dc0R926.

If still not being clear, then documentation have to be improved. Any suggestion?